### PR TITLE
Update the DRF version requirement to match edx-platform

### PR DESCRIPTION
**Pull request by andy-armstrong**
Use the same forked version of DRF in the submissions library as edx-platform.

### CODE REVIEW

**Comment by andy-armstrong** [comment]
@dianakhuang @macdiesel please review.

**Comment by dianakhuang** [comment]
:+1: Hooray.

File: requirements.txt [hunk 1]
```diff
@@ -2,6 +2,8 @@ dogapi==1.2.1
 django>=1.8,<1.9
 django-extensions==1.5.9
 django-model-utils==2.3.1
-djangorestframework>=3.1,<3.2
+# Use the same cherry-picked repo as edx-platform
+#djangorestframework>=3.1,<3.2
```
Comment (cahrens) [inline requirements.txt]: Did you mean to leave in the commented out line (6)?

Comment (andy-armstrong) [inline requirements.txt]: Yes, that's how it was done in edx-platform so I just copied it over. I can remove it if you prefer.

**Comment by macdiesel** [comment]
@andy-armstrong thanks for taking care of this. :+1: :cake:

**Pull request merged**